sw/device/lib/crypto/drivers/BUILD

```diff
@@ -10,20 +10,22 @@ cc_library(
     name = "hmac",
     srcs = ["hmac.c"],
     hdrs = ["hmac.h"],
+    target_compatible_with = [OPENTITAN_CPU],
```

Comment:
This cpu constraint _shouldn't_ be necessary (it currently is however, because abs_mmio erroneously has this constraint too).  By removing this constraint, we can write host-based unit tests for device-side code.

Its ok to leave this in now.  We'll remove this constraint (see #11416) after migrating fully to bazel and eliminating meson.

Reply:
I had to resolve a merge conflict anyway, so I went ahead and removed this requirement for both `abs_mmio` and the two drivers. You're correct, it still builds!